stable/wordpress/templates/deployment.yaml

```diff
@@ -28,21 +28,47 @@ spec:
         - name: MARIADB_ROOT_PASSWORD
           valueFrom:
             secretKeyRef:
+            {{- if .Values.mariadb.enabled }}
               name: {{ template "mariadb.fullname" . }}
               key: mariadb-root-password
+            {{- else }}
```

Comment:
according to the docs (https://github.com/bitnami/bitnami-docker-wordpress#environment-variables) if you're using an existing database, the root password is not needed

Reply:
While I see your point on this one, I don't entirely agree with it. While the root password is only used for DB creation if you had a DB server and had not created the database you would still want to use this feature.
The currently functionality allows for wordpress to use the mysql client to create the database on a remote server.  I think removing that functionality would be a step backwards. If someone does not want to use this they can enter a bogus password.

Reply:
Okay, fair point. I do think this can be further improved by making this optional (e.g. if externalDatabase.rootPassword)